### DISCUSSION TITLE
Verifier is in Processing for last 10 days

### DISCUSSION THREAD

**Discussion by vaibonfire** [Web3Auth/Web3Auth#623] [Help]
Team, 

I have applied for the verifier to be in mainnet but it is in Processing for the last 10 days **bonfire-real-estate**. Please suggest if we need to add any changes to the verifier.

Thanks,
Vai

**Comment by rinzler99** [comment]
Hey, Thanks for reaching out @tainguyentt will help here.

**Comment by tanguyenvn** [comment]
Hi @vaibonfire, we checked and found that we have deployed it but somehow it's stuck at status "Deployed" and not transition to "Live". Several verifiers are affected and we're investigating the cause.
In the mean time, can you help to submit another verifier with same config?

**Reply by vaibonfire** [reply]
Thanks @rinzler99 and @tainguyentt. I have created a new verifier "bonfire-capital" with same config.
